final_task/rss_reader/items.py

```diff
@@ -15,13 +15,20 @@ class Item:
     img_links: List[str]
 
 
-def print_items(items):
-    """ Print all items in stdout.
+@dataclass
+class ItemGroup:
+    feed: str
+    items: List[Item]
+
 
-    :param items: just all items
-    :type items: list of Item
+def print_item_group(item_group):
```

Comment:
It is more logical to make this a method of ItemGroup class.

Reply:
done